src/App.jsx

```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {!selectedGood ? (
+          'No goods selected'
+        )
+          : (
+            <>
+              {`${selectedGood} is selected`}
+              <button
+                onClick={() => setSelectedGood('')}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </>
+          )
+        }
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            good !== selectedGood ? (
+              <tr key={good} data-cy="Good">
```

Comment:
Don't repeat yourself. Don't use separate blocks for selected and not selected goods.